src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -180,6 +180,9 @@ public IJsonTypeInfoResolver TypeInfoResolver
             }
         }
 
+        // Needed since public property is RUC.
```

Comment:
```suggestion
        // Needed since public property is RequiresUnreferencedCode.
```

otherwise I think we won't know what that means in 3 years